From f77fb06dc5d9f7930da9c757d6916f1149dc42bd Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sun, 29 Mar 2020 13:55:53 +0200
Subject: [PATCH] Fix build with Qt 5.15

Summary:
QPainterPath is no longer included via qtransform.h (since
5.15.0-beta2, 50d2acdc93b4de2ba56eb67787e2bdcb21dd4bea in qtbase.git).

Test Plan: Builds

Reviewers: #calligra:_3.0, danders

Differential Revision: https://phabricator.kde.org/D28403
---
 src/libs/ui/kptganttitemdelegate.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libs/ui/kptganttitemdelegate.cpp b/src/libs/ui/kptganttitemdelegate.cpp
index 04217da4..d2f3e6d8 100644
--- a/src/libs/ui/kptganttitemdelegate.cpp
+++ b/src/libs/ui/kptganttitemdelegate.cpp
@@ -28,6 +28,7 @@
 #include <QModelIndex>
 #include <QApplication>
 #include <QPainter>
+#include <QPainterPath>
 #include <QLocale>
 
 #include <KLocalizedString>
-- 
GitLab

